### ISSUE TITLE
Substring weighting too low for long path-names

### ISSUE THREAD

**Issue by euank** [euank/pazi#23]
Reproduction:

```
$ mkdir "/tmp/this is a pretty long name" && cd "/tmp/this is a pretty long name" && cd
$ z long # exit 1, no matches

**Comment by euank** [comment]
this also might be a job for a higher weighted "sub-match" thing which splits on spaces/dots/dashes and matches more strongly against things within there

**Comment by LinuxMercedes** [comment]
Probably closed by #39 !
